docs/user/editions/budgie/configuration.md

```diff
@@ -241,50 +241,48 @@ Click one of the images displayed on the right to choose a pre-installed backgro
+Desktop Icons are enabled by default when Budgie is installed for the first time. To check or toggle this setting, open the Budgie Desktop Settings app by selecting "Budgie Desktop Settings" from the Budgie Menu, or from the pop-up menu when right-clicking on a blank spot on the desktop.
 
-Alternatively, you can open the Budgie Menu and then click "Desktop" in the System Settings category.
+In the Budgie Desktop Settings app, click Desktop in the left-hand sidebar - Desktop Icons is the first toggle switch on that screen.
 
-![Budgie Desktop Icons](configuration/budgie-desktop-icons.jpg)
+![Budgie Desktop Icons](configuration/budgie-desktop-settings-desktop-section.jpg)
 
-You may now add things to your Desktop. This can be done by dragging an item onto the Desktop or by right clicking an icon and selecting Copy to / Move to Desktop. Alternatively, open the Budgie Menu and click on "Files" and then move the files you want to the Desktop folder (as shown in the image below). Please note, the Desktop folder is not visible in "Files" until you've enabled the Desktop icons.
+To place files on your desktop, you can either drag them directly onto the Desktop, or move the files you want to the Desktop folder within the Nemo file manager app (as shown in the image below).
 
-![Budgie Desktop Folder](configuration/budgie-nautilus-desktop-folder.jpg)
+![Budgie Desktop Folder](configuration/budgie-nemo-desktop-folder.jpg)
 
 ## Workspaces
 
-Budgie enables the switching of workspaces via the Workspace Switcher. You can enable workspaces in Budgie by adding the Workspace Switcher applet to Raven.
+Budgie enables the switching of workspaces via the Workspace Switcher. You can enable workspaces in Budgie by adding the Workspace Switcher applet to your panel.
 
-Open Raven using the shortcut `Super + A` or use your mouse to open Raven in the far right of the main Budgie panel.
+To add this applet, open the Budgie Control Center by selecting "System Settings" from either the Budgie Menu, or by right-clicking on a blank spot on the desktop. Once in the Budgie Control Center, in the left-hand sidebar, select the panel to which you want to add the Workspace Switcher (choose the default Bottom Panel if you don't want to add more panels at this time).
 
-Click the Raven settings 'cog' icon
+![Budgie Desktop Settings Panel Section](configuration/budgie-desktop-settings-bottom-panel.jpg)
 
-![Raven Settings Icon](configuration/raven-settings-cog.jpg)
-
-Alternatively, you can open the Budgie Menu and then click "Bottom Panel" in the System Settings category.
-
-![Budgie Panel Menu](configuration/budgie-panel-menu.jpg)
+Click the + button next to 'Add applet' on the right side of the window. This will open the list of available applets - scroll down until you find `Workspace Switcher`, select it, and then click the 'Add' button in the top-right of the window.
```

Comment:
use quotes around "Workspace Switcher".
Backticks are reserved for package names, code, text entry, and so on